ppb/systems.py

```diff
@@ -129,6 +129,9 @@ def __init__(self, resolution=default_resolution, window_title: str="PursuedPyBe
         self.resources = {}
         self.window = None
         self.window_title = window_title
+        self.pixel_ratio = 80  # TODO: Default this somewhere sensible.
```

Comment:
Can we leave this as `None`? It's pulled from camera as soon as rendering begins.